Formula/t/traefik@2.rb

```diff
@@ -0,0 +1,69 @@
+class TraefikAT2 < Formula
+  desc "Modern reverse proxy"
+  homepage "https://traefik.io/"
+  url "https://github.com/traefik/traefik/releases/download/v2.11.2/traefik-v2.11.2.src.tar.gz"
+  sha256 "48a8b8a247991db02d4e3f9aac6db512eaea265d22f0f407ccb038d72d02a106"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    regex(/^v?(2\.\d+\.\d+)$/i)
+  end
+
+  keg_only :versioned_formula
+
+  # https://doc.traefik.io/traefik/deprecation/releases/
+  deprecate! date: "2025-04-30", because: :unmaintained
```

Comment:
Support will end then, it's already deprecated 
```suggestion
  disable! date: "2025-04-30", because: :unmaintained
```